pkg/rollout/revision.go

```diff
@@ -0,0 +1,57 @@
+package rollout
+
+import (
+	"google.golang.org/api/run/v1"
+)
+
+// Annotations name for information related to the rollout.
+const (
+	StableRevisionAnnotation    = "rollout.cloud.run/stableRevision"
```

Comment:
Any ideas of another annotation key? Or is this fine?

Reply:
this is good for now.